internal/bminventory/inventory.go

```diff
@@ -2543,6 +2543,17 @@ func (b *bareMetalInventory) updatePlatformParams(params installer.V2UpdateClust
 	if params.ClusterUpdateParams.Platform != nil && common.PlatformTypeValue(params.ClusterUpdateParams.Platform.Type) != "" {
 		updates["platform_type"] = params.ClusterUpdateParams.Platform.Type
 		updates["platform_is_external"] = swag.BoolValue(params.ClusterUpdateParams.Platform.IsExternal)
+		if *params.ClusterUpdateParams.Platform.Type == models.PlatformTypeExternal {
```

Comment:
sometime we use common.PlatformTypeValue(params.ClusterUpdateParams.Platform.Type) and sometimes we use  *params.ClusterUpdateParams.Platform.Type - why?

Reply:
`common.PlatformTypeValue()` returns an string empty string when `Platform.Type` is `nil`.